deps/couchbeam-0.8.0/Makefile

```diff
@@ -3,9 +3,7 @@ ROOT = ../..
 REBAR = $(ROOT)/utils/rebar/rebar
 DIALYZER = dialyzer
 
-EBINS = $(shell find $(ROOT)/core -maxdepth 2 -name ebin -print) \
-	$(shell find $(ROOT)/deps -maxdepth 2 -name ebin -print)
-PA = $(foreach EBIN,$(EBINS),-pa $(EBIN))
+PA =
```

Comment:
`src/couchbeam_attachments.erl:11: can't find include lib "couchbeam/include/couchbeam.hrl"`

`-include_lib` uses `-pa` I beleive. You should add them back and maybe remove some `-I`s.